internal/store/persistentvolume.go

```diff
@@ -219,6 +221,13 @@ func persistentVolumeMetricFamilies(allowAnnotationsList, allowLabelsList []stri
 				case p.Spec.PersistentVolumeSource.CSI != nil:
 					csiDriver = p.Spec.PersistentVolumeSource.CSI.Driver
 					csiVolumeHandle = p.Spec.PersistentVolumeSource.CSI.VolumeHandle
+
+					for k, v := range p.Spec.PersistentVolumeSource.CSI.VolumeAttributes {
+						if len(attributes) != 0 {
+							attributes += ","
+						}
+						attributes += fmt.Sprintf("%s=%s", k, v)
+					}
```

Comment:
This seems very much unbounded and would require a similar protection mechanism as the one we have for labels and annotations to avoid cardinality explosions.

Reply:
I tried to reduce the possible cardinality of labels. Please check if this would work better. Labels are already unique per PV since one of the labels is the name of the volume and I expect both the mounter and the map options to rarely be changed